.github/workflows/nexus-operator-integration-checks.yml

```
@@ -62,9 +62,15 @@ jobs:
 
       - name: Test Packages
         run: go test ./pkg/... -count=1 -coverprofile=coverage.txt -covermode=atomic
-
+      - name: Validate codcov yaml file
+        run: cat codecov.yml | curl --data-binary @- https://codecov.io/validate
```

I think we can simplify this to spawn a single process. We might also want to add verbosity so we can check what happened in case something goes wrong with the connection itself:

```suggestion
        run: curl -vvv --data-binary @codecov.yml https://codecov.io/validate
```

---

Ah yes, this should work. I just copied the command from [here](https://codecov.io/gh/Kaitou786/nexus-operator/settings/yaml)